jest.config.base.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
   transform: {
-    '^.+\\.tsx?$': ['@swc/jest']
+    '^.+\\.(t|j)sx?$': ['@swc/jest']
```

Comment:
so we transpile JS files as well, particularly for the possible files in `transformIgnorePatterns` -> `node_modules/ky`